### DISCUSSION TITLE
Loading Effect Laravel Jetstream Inertia Js

### DISCUSSION THREAD

**Discussion by faizeee** [inertiajs/inertia#384] [Help] [answered]
Hi there i am try to add Loading effect with basic setting but it is not hiding or completing it struck at 99%. here are my settings

import { InertiaApp } from '@inertiajs/inertia-vue';
import { InertiaForm } from 'laravel-jetstream';
import { InertiaProgress } from '@inertiajs/progress/src';

InertiaProgress.init({
    // The color of the progress bar.
    color: '#dd2238',
    // Whether to include the default NProgress styles.
    includeCSS: true,
    // Whether the NProgress spinner will be shown.
    showSpinner: false,
});

**Comment by claudiodekker** [comment]
Could you give us more information about the request itself? And what version(s) of Inertia/the Inertia adapter are you using?

**Reply by faizeee** [reply]
Request is simple page visit request
iam using latest version of inertia 

-  "@inertiajs/inertia": "^0.3.6",
- "@inertiajs/inertia-vue": "^0.2.4",
-  "laravel-jetstream": "^0.0.3",

**Comment by claudiodekker** [comment, marked as answer]
Alright, so, it's likely you're running into this because of an old bug that's been fixed in more recent versions.

Please try upgrading to the latest `@inertiajs/inertia` (0.8.2), `@inertiajs/inertiajs-vue` (0.5.4) and `@inertiajs/progress` (0.2.4) versions, as listed on https://github.com/inertiajs/inertia/releases and https://github.com/inertiajs/progress/releases. Upgrading `laravel-jetstream` isn't necessary, since it only gets used when you set up your front-end scaffolding initially.

Do note that when making an upgrade like this, there are things in your application that will very likely break due to the (somewhat large) version difference. As such, I recommend you step through through the releases all the way from your application's installed versions up to the current ones, and to apply all of the breaking changes/fixes that are relevant to your application. You can ignore all packages that aren't relevant for you (such as `@inertiajs/inertia-vue3`)

I understand that this is less than ideal, but on the positive side this upgrade doesn't just solve your issue but also adds multiple new features that should make your Inertia code a bit more enjoyable to work with.
